homeassistant/components/hyundai_kia_connect/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "domain": "hyundai_kia_connect",
+  "name": "Hyundai / Kia Connect",
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/hyundai_kia_connect",
+  "requirements": ["hyundai_kia_connect_api==1.18.12"],
```

Comment:
Currently, version 1.31.0 is released. Should this be updated before the release of this integration?

https://pypi.org/project/hyundai-kia-connect-api/1.31.0/